### ISSUE TITLE
mitsuba2 Mitsuba2 variants configuration issue

### ISSUE THREAD

**Issue by ZFaniso** [mitsuba-renderer/mitsuba2#164]
Hi
I have a problem with my mitsuba2 variants. I followed all the steps in the mitsuba2 docs.
step1: I cloned the mitsuba repo using the link : git clone --recursive https://github.com/mitsuba-renderer/mitsuba2
step2: Choose variants and configured using the line...cd <..mitsuba repository..>
cp resources/mitsuba.conf.template mitsuba.conf
I opened mitsuba.conf and the enabled variants are spectral_scalar and spectral_rgb.
Step3: I compiled the system using the suggested commands
step4: I ran Mitsuba using the setpath.sh (source setpath.sh)
Step5: I opened the python jupyter notebook here on the mitsuba2 directory, i imported mitsuba, but now when i want to test it using the examples provided, when i set mitsuba variants (mitsuba_setvariants('scalar_rgb') i get an error which says Attrubute error mitsuba module has no attribute set_varient or when i say mitsuba.variants() to check all available mitsuba variants i get the same error which says mitsuba module has no attribute variants

**Comment by Speierers** [comment]
Hi @ZFaniso ,

Which OS are you on?

Also did you compile mitsuba in a `/build` folder?

**Comment by ZFaniso** [comment]
Hi Speierers
I'm using ubuntu 20.04. no i didn't build it in a build folder because i couldn't find it in my mitsuba2 cloned folder. My mitsuba was situated on the include folder so i took it out so that it can be accessible straight from the directory, then i got confused if i should create the build folder by myself or what.(I saw in the setpath.sh that there's MITSUBA_DIR/dist/python and MITSUBA_DIR/BUILD/dist/python). I will be happy if you can clarify it to me on how should i do it. Basically i compiled everything inside mitsuba2 directory. 




![mitsuba2directory](https://user-images.githubusercontent.com/41281683/83487267-05afed80-a4ab-11ea-973a-d329ba075920.png)

**Comment by Speierers** [comment]
I think it is quite clear in the doc. I recommend you try again from scratch:

```
git clone --recursive https://github.com/mitsuba-renderer/mitsuba2
cd mitsuba2
mkdir build
cd build
cmake  ..
make
```
Let me know if this works for you.

**Comment by ZFaniso** [comment]
Ok it seems like it worked because i didn't get errors ,thanks. Now i opened my jupyter notebok then typed import mitsuba to test i find a python error which says 
ImportError: The 'mitsuba' native modules could not be imported. You're likely trying to use Mitsuba within a Python binary (/home/zimbini/anaconda3/bin/python) that is different from the one for which the native module was compiled (/usr/bin/python3.8). which is true because i have python 3.7.
So i exported python path=/usr/bin/python3.8 in the bash script but it doesn't work. Can you advice on this.

**Comment by Speierers** [comment]
you will need to recompile mitsuba with `python3.8` if this is the one you want to use.
You can also set the path to a python binary in `cmake` or `ccmake`.

I will close this as it is not an issue.

**Comment by ZFaniso** [comment]
yes you can close thanks.



>>> Sebastien Speierer <notifications@github.com> 06/02/20 4:03 PM >>>

 Closed #164.
